README.md

```diff
@@ -242,3 +244,81 @@ const result = await myConsumer.handlerSpy.waitForMessageWithId('1')
 expect(result.processingResult).toEqual('consumed')
 ```
 
+## Payload Offloading
+Payload offloading allows you to manage large message payloads by storing them in external storage, bypassing any message size restrictions imposed by queue systems.
+
+### Configuration
+
+1. **Install a payload store implementation (S3 in example):**
+   ```bash
+   npm install @message-queue-toolkit/s3-payload-store
+   ```
+
+2. **Configure your setup:**
+    ```typescript
+    import {S3} from '@aws-sdk/client-s3'
+    import {S3PayloadStore} from '@message-queue-toolkit/s3-payload-store'
+    import {PayloadStoreConfig} from "@message-queue-toolkit/core"
+    
+    const s3Client = new S3({
+        region: 'your-s3-region',
+        credentials: {
+            accessKeyId: 'your-access-key-id',
+            secretAccessKey: 'your-secret-access-key'
+        }
+    })
+    
+    const payloadStoreConfig = {
+        store: new S3PayloadStore(
+            {s3: s3Client},
+            {
+                bucketName: 'your-s3-bucket-name',
+                keyPrefix: 'optional-key-prefix'
+            }
+        ),
+        messageSizeThreshold: 256 * 1024 // 256KB
```

Comment:
Can we expose limits, compatible with SNS and SQS as constants from according packages?

Reply:
Done: https://github.com/kibertoad/message-queue-toolkit/pull/167/commits/fff4268f892dedb12da86e5aa673295c5f041648